manifests/kots-config.yaml

```diff
@@ -55,37 +55,38 @@ spec:
             repl{{    $_ := set $result "launchDarklyClientSideId" "NjIxNjNmMTYyZDliOTUxNGI4MzI1NWM2" }}
             repl{{ end }}
             repl{{ toJson $result }}
-        - name: kotsEncryptionSecretKey
+        - name: autogeneratedEncryptionSecretKey
           title: Customer Package Values
           type: textarea
           hidden: true
           readonly: false
           value: '{{repl RandomString 32 "[A-Za-z0-9]" }}'
-        - name: kotsVarnishDebugSecret
+        - name: autogeneratedJwtApiSecret
           title: Customer Package Values
           type: textarea
           hidden: true
           readonly: false
           value: '{{repl RandomString 16 "[A-Za-z0-9]" }}'
-        - name: kotsJwtApiSecret
+        - name: autogeneratedVarnishDebugSecret
           title: Customer Package Values
           type: textarea
           hidden: true
           readonly: false
           value: '{{repl RandomString 16 "[A-Za-z0-9]" }}'
-        - name: kotsVarnishPurgeSecret
+        - name: autogeneratedVarnishPurgeSecret
           title: Customer Package Values
           type: textarea
           hidden: true
           readonly: false
           value: '{{repl RandomString 16 "[A-Za-z0-9]" }}'
-        - name: internalRedisPassword
+        - name: autogeneratedInternalRedisPassword
           title: password used for the internal redis
           type: textarea
           hidden: true
           readonly: false
           value: '{{repl RandomString 16 "[A-Za-z0-9]" }}'
-        - name: projectId
+
+        - name: derivedSelfHostedGCPProjectId
```

Comment:
Why do we include the derived prefix in some properties? 🤔 It sounds to me as if they were "computed" values. Values obtained from other ones, no?

Reply:
Finally it was a misunderstanding and the current name was great.